{{cookiecutter.profile_name}}/lsf_status.py

```diff
@@ -125,20 +125,29 @@ def _kill_job(self):
     def _query_status_using_log(self) -> str:
         try:
             log_tail = self._get_tail_of_log_file()
+        except FileNotFoundError:
+            print("Log file {} not found".format(self.outlog), file=sys.stderr)
+            return self.FAILED
```

Comment:
My only doubt in this PR is here. I am afraid of us erroneously assuming the log file does not exist due to filesystem latency. I wonder if we do not find the log file, then we should wait for the duration of `latency_wait` and retry?

I have another doubt. Before, we were returning `RUNNING` because we assumed the log file should always be created. Isn't this the case? We returned `RUNNING` as it would be eventually created, and then we would be able to read from it. Could you give me an example when LSF does not create log files for jobs? Maybe when LSF is configured to email them, but then this whole feature of looking at log files if jobs disappear won't work anyway... I think bottom line is: for this feature to work, LSF has to necessarily create log files. And if this is true, then not finding the log file is due to some sort of latency (either due to LSF or filesystem), so we should just wait. But I think you have a counter example to this, as you got trapped in an infinite loop.

Reply:
I can add in a latency wait I guess. 

> I have another doubt. Before, we were returning RUNNING because we assumed the log file should always be created. Isn't this the case? 

No, it isn't. I was seeing this a lot recently with UNKWN jobs and their log files not being created. And latency has nothing to do with it. These jobs were unknown overnight and then in the morning I went to look at their log files and they don't exist.

Reply:
All good for me with your new change where you added the latency wait! From my POV, we can merge!